Wiki/Addnodes.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Gridcoin Addnodes
+layout: wiki
+---
+
+
+# Gridcoin addnodes
+### This is a list of nodes for your gridcoinresearch.conf file to make syncing much smoother.  
+
+You can also add the node with the command: "addnode <node>
+<add|remove|onetry>" in the debug console.
+
+Last updated: Mon Oct 28 13:54:39 UTC 2019
+
+Windows: `C:\Users\<username>\AppData\Roaming\GridcoinResearch`
+
+macOS:   `/Users/<username>/Library/Application Support/GridcoinResearch`
+
+Linux:   `/home/<username>/.GridcoinResearch`
+
+### Mainnet
```

Comment:
This does not format correctly on build. | should technically become a table but this does not happen currently. This seems limited to this page as the scraper displays correctly.

Reply:
Fixed that, don't know how I missed it. The tables have to be separated by newlines on both sides of the tables and these weren't. I also noticed that the table had something that was supposed to span the column which doesn't work on kramdown so I changed it to be headers above the tables